tests/test_plot_interaction.py

```diff
@@ -153,8 +153,11 @@ def test_interaction_attached():
         # exclude functions that do not support interaction
         if function[0] in ["context", "custom_subplots"]:
             continue
-
-        ax = function[1](signal)
+        if function[1] == pf.plot.spectrogram:
+            with pytest.warns(UserWarning, match="Using only the first"):
+                ax = function[1](signal)
```

Comment:
using `function[1](signal[0])` would solve the issue more explicitly

Reply:
yes, changed that